File: app/assets/javascripts/partners/partner_helps.js
$(document).ready(function() {
$("#form_save_partner_help").validate({
        rules: {
            "partner_help[question]":{
                required: true
            },
            "partner_help[details]":{
                required: true
            },
            "partner_help[first_name]":{
                required: true
            },
            "partner_help[last_name]":{
                required: true
            },
            "partner_help[area_code]":{
                required: true,
                number: true,
                minlength: 3,
                maxlength: 3
            },
            "partner_help[phone1]":{
                required: true,
                number: true,
                minlength: 3,
                maxlength: 3
            },
            "partner_help[phone2]":{
                required: true,
                number: true,
                minlength: 4,
                maxlength: 4
            },
            "partner_help[email]":{
                required: true,
                email: true
            }
        },
        messages: {
            "partner_help[question]":{
                required: "Please enter question"
            },
            "partner_help[details]":{
                required: "Please enter details"
            },
            "partner_help[first_name]":{
                required: "Please enter first name"
            },
            "help[last_name]":{
                required: "Please enter last name"
            },
            "partner_help[area_code]":{
                required: "required"
            },
            "partner_help[phone1]":{
                required: "required"
            },
            "partner_help[phone2]":{
                required: "required"
            },
            "partner_help[email]":{
                required: "Please enter email"
            }
        }
    });
});

File: app/admin/partner_helps.rb
ActiveAdmin.register PartnerHelp do
	menu :priority => 12, :label => "PartnerHelp"
	filter :user
  filter :question

  index do
    column :question
    column :details
    default_actions
  end

  show :title => "PartnerHelp" do |help|
    attributes_table do
      row :id
      row :question
      row :details
      row :partner_master_id
      row :first_name
      row :last_name
      row :area_code
      row :phone1
      row :phone2
      row :email
    end
    active_admin_comments
  end

  form do |f|
    f.inputs "PartnerHelp" do
      f.input :question
      # f.input :tenant, :as => "select", :collection => Tenant.active_tenants_only.map{|t| [ "#{if t.subdomain then t.subdomain else "User not set" end}", t.subdomain ]}#, {prompt: 'Select Tenant', :selected => "#{tenant}"}
      f.input :details
      f.input :first_name
      f.input :last_name
      f.input :area_code
      f.input :phone1
      f.input :phone2
      f.input :email
    end
    f.actions
  end
  controller do
    before_filter :reset_tenant

    skip_before_filter :decide_redirection

    def reset_tenant
      ActsAsTenant.current_tenant = nil
    end
  end
  
end
